snuba/datasets/errors.py

```diff
@@ -155,6 +155,14 @@ def __init__(self) -> None:
             sample_expr="event_hash",
             ttl_expr="timestamp + toIntervalDay(retention_days)",
             settings={"index_granularity": "8192"},
+            required_deletion_columns=[
+                "event_id",
+                "project_id",
+                "group_id",
+                "timestamp",
+                "deleted",
+                "retention_days",
+            ],
```

Comment:
See comment below for more context, but by following the same logic from the definition in `EventsDataset`, it's unclear to me where this column set comes from considering this table layout is considerably different and this table contains far less nullable fields.

Reply:
Indeed there are more required columns to add.
Unfortunately I cannot test the replacement on that table till https://github.com/getsentry/snuba/pull/774 so I could not see that.